### ISSUE TITLE
Combinding two PBPK models - sim results different

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#1209]
[CombinedModels_DifferentResults.zip](https://github.com/Open-Systems-Pharmacology/MoBi/files/13366132/CombinedModels_DifferentResults.zip)

**Comment by Yuri05** [comment]
Results between which simulations were different?

**Comment by PavelBal** [comment]
"Bahrami 2007, 100 mg SD_new" vs "Combined_IndivBahrami", output "Fluvoxamine Peripheral Venous Blood"

![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/25903040/1bf9c17f-345d-4e0d-8553-d7458fa718e2)

**Comment by PavelBal** [comment]
Hm, not even sure this is worth documenting. Actually it makes totally sense, the same would happen if you combine two administrations in PK-Sim, right?

**Comment by Yuri05** [comment]
> Actually it makes totally sense, the same would happen if you combine two administrations in PK-Sim, right?

yes

**Comment by rwmcintosh** [comment]
This can be closed right @Yuri05

**Comment by Yuri05** [comment]
> This can be closed right @Yuri05

Yes. The only open question remains the documentation discussion (https://github.com/Open-Systems-Pharmacology/MoBi/issues/1209#issuecomment-1847344294) - but we could create another issue for that.
